SU-SWS/stanford_profile
# D8CORE-2110: Remove news views paragraph type.

# READY FOR REVIEW

# Summary
- Removes the news view paragraph type and all associated code/tests

# Needed By (Date)
- asap

# Urgency
- top

# Steps to Test

1. Check out this branch
2. Do a full install (`blt di`)
3. Validate no paragraph type exists

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- D8CORE-2110
- https://github.com/SU-SWS/stanford_news/pull/78

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
